system/Validation/Rules.php

```diff
@@ -209,7 +209,7 @@ public function not_in_list(?string $value, string $list): bool
     }
 
     /**
-     * @param mixed $str
+     * @param object|array|string|int|float|null $str
```

Comment:
What about `bool`?

Reply:
Oh, I miss this.
I have been maked up.
Appreciated that!